README.md

```diff
@@ -21,7 +21,8 @@ Before you begin deploying 1Password SCIM Bridge, review the [Preparation Guide]
 
 Containers as a service (CaaS) can simplify your deployment by using the built-in tools of the CaaS for DNS and certificate management. This gives you an easy, low-cost SCIM bridge with minimal infrastructure management requirements.
 
-- [Azure Container Apps](/azure-container-apps)
+- [Azure Container Apps](https://support.1password.com/scim-deploy-azure/)
+  - [Container Apps advanced references](/azure-container-apps)
```

Comment:
I feel like we should say something besides "advanced references". Maybe "advanced topics"? Perhaps Jacob will have a better idea. 😂

Reply:
Again, hoping during Jacob's review of the documentation at large he will have a better wording selection here.